docs/readme.md

```diff
@@ -0,0 +1,25 @@
+# Taxonomy
+
+## Unclassified Tree
+
+Nodes are deemed *unclassified* when their *scientific name* contains (ignore case) the word `unclassified`; these generate the **Unclassified** tree.
+
+## Environmental Samples Tree
+
+The **Environmental Samples** tree is generated by the nodes whose *scientific name* has prefix (ignore case) `environmental samples`.
+
+# File structure
+
+Each relation is stored as two `csv` files (in and out): first column source, second column target. We have the following relations
+
+1. `parent` (the tree structure)
+2. `scientific_name`
+3. `rank`
+
+For all these entities we use their obvious `String` representation.
+
+Of course, we have one set of this files for each of the trees we provide (full, good, unclassified, environmental samples).
+
+## Tree mappings
+
+The inclusions of the other trees in the full one could be represented in the same way as with any other relation, the source and target being the same ID. Alternatively, we could represent the corresponding membership predicate.
```

Comment:
I don't understand the *source and target being the same ID* either, but I guess I will when I understand what you meant with the relations and the csv files above.